doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -135,6 +137,130 @@ The string "ISO8601" as type for attributes and metadata is also supported. The
 
 [Top](#top)
 
+## `dateModified` and `dateCreated` attributes
+
+Section "Attribute names restrictions" establishes that system/builtin attribute names cannot be used as user attribute names. Thus,
```

Comment:
The section ...

Reply:
Fixed in1ab8d0a878a633c982dbe2f016168e73233d3f1b